.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "ci"]
 	path = ci
-	url = https://github.com/fabiand/traviskube
+	url = https://github.com/mmazur/traviskube
```

Comment:
I understand there are bugs. Please paste reference or describe why we are using our fork.

Reply:
My changes are very trivial, and update here and there. The only reason is the RH Summit. Fabian won't be looking at low prio PRs for some time and I didn't want to get stuck on this. As soon Fabian is back to his usual responsive self I'm planning on getting my changes merged and reverting this back to his repo.

Reply:
Maybe we can move it under kubevirt org